spec/nepali_calendar_spec.rb

```diff
@@ -158,6 +158,23 @@
       d1 = NepaliCalendar::AdCalendar.bs_to_ad(2076, 7, 15).end_of_month
       expect(d1).to eq(Date.parse('Sat, 30 Nov, 2019'))
     end
+
+    describe '.to_date' do
+      it 'returns a Date object for a valid date string' do
```

Comment:
dude what are you doing? 

.to_date is supposed to convert instance of NepaliCalendar::AdCalendar to Date, 
Not the String to Date

Reply:
instead you should go on like:

```
# year, month and day should be as random as possible to increase the probablity of encountering error if there is a situation
ad_calendar = NepaliCalendar::AdCalendar.new(year, month, day)
date = ad_calendar.date
# carry on with the tests
```

Reply:
i was trying to solve this issue https://github.com/lalusaud/nepali_calendar/issues/3